store/localstore/goleveldb/goleveldb.go

```diff
@@ -55,8 +54,13 @@ func (d *db) NewBatch() engine.Batch {
 	return b
 }
 
-func (d *db) Seek(startKey []byte) (engine.Iterator, error) {
-	return d.DB.NewIterator(&util.Range{Start: startKey}, nil), nil
+func (d *db) Seek(startKey []byte) ([]byte, []byte, error) {
+	iter := d.DB.NewIterator(&util.Range{Start: startKey}, nil)
+	defer iter.Release()
+	if ok := iter.First(); !ok {
+		return nil, nil, errors.Trace(engine.ErrNotFound)
+	}
+	return iter.Key(), iter.Value(), nil
```

Comment:
goleveldb's document indicates that it is safe to read but unsafe to modify. I am going to clone slices before return, anyway.